### PR TITLE
rdrf #1393 Management command added for setting the registry version.

### PR DESCRIPTION
**Pull request by jithendralal**
- The management command to display the current registry version and to accept an input of the new version from the user, to be set to the registry

### CODE REVIEW

File: rdrf/rdrf/management/commands/set_registry_version.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+from django.core.management import BaseCommand
+from rdrf.models.definition.models import Registry
+
+
+class Command(BaseCommand):
+    help = "Set the registry version"
+
+    def handle(self, *args, **options):
```
Comment (id2359) [inline rdrf/rdrf/management/commands/set_registry_version.py:8]: Make more specific as we do have the possibility of having more that one registry on a site.

File: rdrf/rdrf/management/commands/set_registry_version.py [hunk 2]
```diff
@@ -6,10 +6,14 @@ class Command(BaseCommand):
...
-        print("\nATTENTION: You are about to change the version of Registry")
-        for registry in Registry.objects.all():
-            current_version = registry.version or "Not set"
-            print(f"\nCurrent version of {registry.name} is: {current_version}")
-            new_version = input("Enter new version: ")
-            registry.version = new_version
-            registry.save()
+        print("\nThe current registry versions:\n")
+
+        registries = Registry.objects.all().values("name", "code", "version").order_by("code")
```
Comment (id2359) [inline rdrf/rdrf/management/commands/set_registry_version.py:11]: Better to use same non-interactive pattern ( option parser) as for the other commands.

Comment (jithendralal) [reply rdrf/rdrf/management/commands/set_registry_version.py]: modified as suggested

File: rdrf/rdrf/management/commands/set_registry_version.py (hunk 1 above)
Comment (jithendralal) [reply rdrf/rdrf/management/commands/set_registry_version.py:22]: modified as suggested

**Pull request merged**